go-controller/pkg/ovn/loadbalancer/loadbalancer.go

```diff
@@ -169,16 +186,11 @@ func EnsureLBs(nbClient libovsdbclient.Client, externalIDs map[string]string, LB
 		return err
 	}
 
-	_, err = libovsdbops.TransactAndCheckAndSetUUIDs(nbClient, lbs, ops)
+	_, err = libovsdbops.TransactAndCheckAndSetUUIDs(nbClient, desired, ops)
```

Comment:
@jcaamano now that we're not caching separately, can we switch this to just `Transact()`?